src/SerilogWeb.Classic.WebApi/Classic/WebApi/HttpActionContextExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http.Controllers;
+
+namespace SerilogWeb.Classic.WebApi
+{
+    internal static class HttpActionContextExtensions
+    {
+        public static void StoreWebApInfoInHttpContext(this HttpActionContext actionContext)
```

Comment:
This one should probably be marked as `internal` for now ... (although it's already in an `internal` class, so I'm not really sure what this would change :p

Reply:
No problem, I can change that